### PR TITLE
feat: enable row timestamp metrics with environment variable

### PR DESCRIPTION
**Pull request by tustvold**
These metrics are very high-cardinality, 60 per table, which can causes issues for some monitoring systems. This PR makes it so that they are only enabled on a per-table basis with an environment variable

### CODE REVIEW

File: tests/common/server_fixture.rs [hunk 1]
```diff
@@ -442,7 +458,7 @@ impl TestServer {
         };
     }
 
-    /// Create a connection channel for the gRPR endpoing
```
Comment (tustvold) [inline tests/common/server_fixture.rs:445]: Endpoing :laughing:

File: tests/end_to_end_cases/metrics.rs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+use crate::common::server_fixture::ServerFixture;
+use crate::end_to_end_cases::scenario::Scenario;
+
+#[tokio::test]
+pub async fn test_row_timestamp() {
```
Comment (alamb) [inline tests/end_to_end_cases/metrics.rs:5]: 👍

**Review by alamb — APPROVED** [review]
Looks good to me -- I was going to suggest we document `INFLUXDB_IOX_ROW_TIMESTAMP_METRICS` somewhere but then again I am not sure where we would do so.

If we wanted to make it a proper setting, we would probably want to simply make it a real field on `DatabaseRules` but that seems overkill at this point

File: tests/common/server_fixture.rs (hunk 1 above)
Comment (sunng87) [reply tests/common/server_fixture.rs:445]: gPRP 😂

Comment (tustvold) [reply tests/common/server_fixture.rs:445]: Oh lol - :laughing:

**Pull request merged**